orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -65,6 +63,9 @@ def apply(self):
         # TODO change this to custom stopwords
         if self.remove_stpwrds:
             sw = 'english'
+        else:
+            sw = None
```

Comment:
I prefer simple conditional value expressions like this one with a ternary operator:
```py
sw = 'english' if self.remove_stpwrds else None  # sic!
```
Four lines vs. one for something trivial.